starwars/src/Components/StarCharacters.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from "react";
+import {Card, CardBody, CardTitle, CardSubtitle, CardText} from 'reactstrap';
+import PropTypes from 'prop-types';
+
+class StarCharacters extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            starChars: {}
+        };
+    }
+
+    componentDidMount() {
+        this.setState({starChars: this.props.starChars})
```

Comment:
There's no need to store `props` data in `state` here nor use a class at all. You can keep this component as a functional component and use `props` data like this:

```
const StarCharacters = props => {
    return (
        <Card className='card'>
            <CardBody>
            <CardTitle>{props.starChars.name}</CardTitle>
            <CardSubtitle>
                <CardText>gender: {props.starChars.gender}</CardText>
                <CardText>hair color: {props.starChars.hair_color}</CardText>
                <CardText>birth year: {props.starChars.birth_year}</CardText>
                <CardText>eye_color: {props.starChars.eye_color}</CardText>
                <CardText>height: {props.starChars.height}</CardText>
                <CardText>mass: {props.starChars.mass}</CardText>
                <CardText>skin color:{props.starChars.skin_color}</CardText>
            </CardSubtitle>
            </CardBody>
        </Card>
    );
}
```